edk2toolext/omnicache.py

```diff
@@ -393,6 +410,8 @@ def get_cli_options():
                        help="Update the Omnicache.  All cache changes also cause a fetch", default=False)
     group.add_argument("--no-fetch", dest="no_fetch", action="store_true",
                        help="Prevent auto-fetch if implied by other arguments.", default=False)
+    group.add_argument("--fetch-jobs", dest="fetch_jobs", type=int,
```

Comment:
Is this the complement to "parallel" above?

Reply:
Yes; it allows the user to override the "reasonable" default of a thread per CPU and force more or less parallelization if needed. I added it for maximum flexibility here; but most implementations will likely just use the default.